assets/queries/cloudFormation/disable_CDN/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "CDN_missing_configuration",
+  "queryName": "CDN_missing_configuration",
```

Comment:
queryName should have a verb to easily identify  the resource and the wrong action, in this case, something like: "CDN Configuration Is Missing"